### ISSUE TITLE
Normalize ajax load event issue

### ISSUE THREAD

**Issue by Neurostep** [Reactive-Extensions/RxJS-DOM#54]
It looks like the normalization process [here](https://github.com/Reactive-Extensions/RxJS-DOM/blob/master/src/ajax.js#L37) is not correct in case of "requestType": "json" at least. I've just tried to use Rx.DOM.getJSON from the latest release and got an issue: in response I've got a string instead of JS object (parsed string with JSON.parse).

sinon fake XMLHttpRequest hides this problem by [using only responseText](https://github.com/cjohansen/Sinon.JS/blob/master/lib/sinon/util/fake_xml_http_request.js#L512) instead of both response & responseText fields.

My suggestion is to update the normalizeAjaxLoadEvent function like this:

````javascript
function normalizeAjaxLoadEvent(e, xhr, settings) {
    var response = ('response' in xhr) ? xhr.response : xhr.responseText;
    response = settings.responseType === 'json' ? JSON.parse(response) : response;
    return {
      response: response,
      status: xhr.status,
      responseType: xhr.responseType,
      xhr: xhr,
      originalEvent: e
    };
  }
````

Thank you.

**Comment by stkb** [comment]
I've just started using rx-dom and run in to the same issue (IE11). `xhr.response` already contains the response as a string and so `normalizeAjaxLoadEvent` doesn't attempt to parse it into an object. Neurostep's fix seems like a good one.

**Comment by mattpodwysocki** [comment]
@Neurostep done!

**Comment by Neurostep** [comment]
@stkb @mattpodwysocki Thanks!
